src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -111,6 +117,10 @@ def check_cluster_failures(self, cluster: ClusterDetails, source: str) -> list[s
             failures.extend(self._check_cluster_policy(cluster.policy_id, source))
         if cluster.init_scripts is not None:
             failures.extend(self._check_cluster_init_script(cluster.init_scripts, source))
+        if cluster.data_security_mode == DataSecurityMode.NONE:
+            failures.append("No isolation shared clusters not supported in UC")
```

Comment:
can we add this error message to [doc](https://github.com/databrickslabs/ucx/docs/assessment.md) as well